action/assert_test.go

```diff
@@ -15,9 +16,10 @@ import (
 )
 
 func TestLocalAssert(t *testing.T) {
+	os.Chdir(t.TempDir())
```

Comment:
It is safe to change to a tempdir always in every single test. This is because, tests might involve creating/manipulating files like `experiment.yaml`. We don't want to mess up in `testdir` folders accidentally.